### PR TITLE
feat(content): additional unarmed weapons

### PR DESCRIPTION
**Pull request by Pr0man**
## Purpose of change
The top line unarmed weapon, which is currently the katar, sadly has cutting damage as its damage type, which is nice to eat through fodder enemies like standard zombies, but already starts to struggle against cops and above, like SWAT zombies to a noticable degree.

## Describe the solution
With the addition of both a piercing option (jamadhar - a two-bladed variation of the katar that is more suited for stabbing than slashing) as well as a bashing option (impact knuckles - a pair of gloves having been reinforced with steel bracing that has a solid plate in front of your knuckles, imagine metal gauntlets). They are created with roughly similar recipes and difficulties, only having slight alterations to reflect the difference in material.

## Describe alternatives you've considered
use your trusty one-twos instead of unarmed weapons, which is now a viable alternative.

## Testing
The weapons have been tested in an entirely unmodded world, at various skill levels (unarmed 0, unarmed 3, unarmed 6 alongside secondary weapon skills) against various zombies and perform roughly on-par with each other, with piercing slightly lagging behind cutting against unarmored targets, and both noticably weaker than impact knuckles on anything highly armored, with a very noticable jump from SWAT Zombies onwards. However, I'm not entirely certain how well balance is struck with the chosen values, The knuckles in particular are difficult to strike in a right balance, as they now deal slightly more damage than fists on the same level of unarmed, starting at 5, but being a tad slower to compensate. This is mainly to overcome high armor targets.

## Additional context
The Jamadhar is a real equivalent of the katar, based in the indian region, Some of them have thicker blades, some have very thin blades, but they are generally pointed forward and not nearly as broad as the classic katars depiction, which allows for puncturing the target instead of slashing at them.
Impact knuckles as such are generally more of a science fiction weapon, similar to power fists, just without electricity and pistons. However, impact bracing for the knuckle region is a real thing on gloves, specifically in metalworking professions, so taking it a step further by instead creating a bracing around a glove to attach some plating is not entirely unreasonable.

### CODE REVIEW

**Comment by VissValdyr** [comment]
Few things to consider:
1. Keep the id close to the final product, so I would keep the actual names as id.
2. They don't spawn anywhere. Katar is the same. We have "ancient/medieval, rare, spawn groups. Check item group art and crafts. Add them there. It should be possible to find them somewhere.

**Comment by VissValdyr** [comment]
Also after every full stop 2 spaces

File: data/json/items/melee/unarmed_weapons.json [hunk 1]
```diff
@@ -133,6 +133,36 @@
...
+  },
+  {
+    "id": "impact_knuckle",
+    "type": "GENERIC",
+    "category": "weapons",
+    "weapon_category": [ "FIST_WEAPONS" ],
+    "symbol": "]",
+    "color": "dark_gray",
+    "name": { "str": "pair of impact knuckles", "str_pl": "pairs of impact knuckles" },
+    "description": "A pair of sturdy steel bracings, inlaid with padding material, which are then covered by hardened steel plates. Can be used to safely deliver punches full force without the risk of harming ones hands.",
```
Comment (chaosvolt) [inline data/json/items/melee/unarmed_weapons.json:157]: Absolutely mental that the check failure prints 8 bloody times for this one slip-up.
```suggestion
    "description": "A pair of sturdy steel bracings, inlaid with padding material, which are then covered by hardened steel plates.  Can be used to safely deliver punches full force without the risk of harming ones hands.",
```

File: data/json/items/melee/unarmed_weapons.json [hunk 2]
```diff
@@ -133,6 +133,37 @@
     "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 12 ] ],
     "flags": [ "UNARMED_WEAPON", "DURABLE_MELEE", "SHEATH_KNIFE" ]
   },
+  {
+    "id": "jamadhar",
+    "type": "GENERIC",
+    "name": { "str": "jamadhar" },
+    "copy-from": "knuckle_katar",
+    "description": "A steel brace with two thinner, pointed blades at the end, designed to primarely stab victims when used to punch.",
```
Comment (chaosvolt) [inline data/json/items/melee/unarmed_weapons.json:141]:
```suggestion
    "description": "A steel brace with two thinner, pointed blades at the end, designed primary to stab victims when used to punch.",
```

Comment (RobbieNeko) [reply data/json/items/melee/unarmed_weapons.json:141]: *primarily

Comment (chaosvolt) [reply data/json/items/melee/unarmed_weapons.json:141]: Whoops, clicked the wrong suggestion from spellcheck, yeah. XD

**Review by chaosvolt — APPROVED** [review]

**Pull request merged**